wherescape/connectors/hubspot/python_hubspot_check_missing_columns

```diff
@@ -0,0 +1,130 @@
+    sql = f"select hubspot_environment, column_name, hubspot_load_table_name, missing_where from {table_name}"  # noqa: E501
+    rows = wherescape_instance.query_target(sql)
+    return rows
+
+
+def hubspot_check_missing_columns():
+    # Main function to check missing Hubspot columns
+    wherescape_instance = WhereScape()
+    hubspot_access_token_voys = wherescape_instance.read_parameter(
+        "hubspot_access_token_voys"
+    )
+    hubspot_instance = Hubspot(hubspot_access_token_voys)
+    environments = ("voys", "vg")
+    table_rows = []
+
+    # Query the hubspot api per environment and object_type and compare with
+    # the wherescape objects.
+    for environment in environments:
+        for object_type in OBJECT_TYPES_TO_CHECK:
+            table_name = "load_hubspot_%s_%s" % (environment, object_type)
+            sql = "SELECT lt_obj_key FROM ws_load_tab WHERE lt_table_name = ?"
+            results = wherescape_instance.query_meta(sql, [table_name])
+
+            if len(results) > 0:
+                lc_obj_key = results[0][0]
+                sql = "SELECT lc_col_name FROM ws_load_col WHERE lc_obj_key = ?"
+                results = wherescape_instance.query_meta(sql, [lc_obj_key])
+                wherescape_columns = [item[0] for item in results]
+                api_response = None
+                try:
+                    api_response = (
+                        hubspot_instance.client.crm.properties.core_api.get_all(
+                            object_type=object_type, archived=False
+                        )
+                    )
+                except ForbiddenException:
+                    logging.info(
+                        f"No access for type {object_type} via the Hubspot api for {environment}"  # noqa: E501
+                    )
+                else:
+                    api_results = api_response.to_dict()
+                    hubspot_columns = []
+                    for result in api_results["results"]:
+                        hubspot_columns.append(result["name"])
+                    missing_in_wherescape, missing_in_hubspot = compare_columns(
+                        wherescape_columns, hubspot_columns
+                    )
+                    table_rows = create_table_rows(
+                        missing_in_wherescape,
+                        missing_in_hubspot,
+                        environment,
+                        table_name,
+                    )
+            else:
+                logging.info(
+                    f"No load table exists for Hubspot object {object_type} and environment {environment} ({table_name})"  # noqa: E501
+                )
+
+    # Pushing the missing columns to the load table
+    wherescape_instance.push_many_to_target(
+        "insert into %s values (?, ?, ?, ?, null, null)"
+        % wherescape_instance.load_full_name,
+        table_rows,
+    )
+
+    # Get the datastore rows to compare to the new load rows
```

Comment:
Maybe if you do this detection before line 115, it's possible to extract/infer the first date that the missing column occurred and to include it in the load table